INSTALL.md

```diff
@@ -513,5 +496,97 @@ $ git submodule update
 
 Then continue with “Crosscompiling from a released tarball” above.
 
+VII. Compiling on Windows with msys2
+-----------------------------------------
+
+TSC can be compiled on Windows using msys2. The msys2 suite provides a convenient Bash shell for Windows, along with an expanding
+repository of prebuilt packages.
+
+
+### Installing and updating msys2 ###
+
+To begin, download and install msys2 from the [official website][3].
+
+After it's installed run any of the shells and execute:
+
+    $ pacman -Syuu
+
+If the base system components are updated, you will be prompted to close the bash windows manually, go ahead and do so.
+
+Now, run either the `MSYS2 MinGW 32-bit` or the `MSYS2 MinGW 64-bit`, depending on which version you are going to build. Note that you
+can download libraries from one shell for the other, but you cannot build for the other architecture.
+
+Now run update again:
+
+    $ pacman -Syuu
+
+
+### Installing the dependencies ###
+
+Once it's finished, install the dependencies:
+
+    $ pacman -S --needed git bison ruby mingw-w64-x86_64-{toolchain,extra-cmake-modules,ruby,mruby,cegui,sfml,libxml++2.6,gperf}
+
+Or, for 32-bit:
+
+    $ pacman -S --needed git bison ruby mingw-w64-i686-{toolchain,extra-cmake-modules,ruby,mruby,cegui,sfml,libxml++2.6,gperf}
+
+#### Optional dependencies ###
+
+The following packages are optional, you don't have to install these if you don't plan to generate installers or documentation:
+
+    $ pacman -S --needed mingw-w64-x86_64-{doxygen,graphviz,nsis,qt-installer-framework-git}
+
+For 32-bit:
+
+    $ pacman -S --needed mingw-w64-x86_64-{doxygen,graphviz,nsis,qt-installer-framework-git}
```

Comment:
qt-installer-framework? What exactly requires it? I thought the TSC installer just used NSIS?

Reply:
As of now, the NSIS generator does not work. So, I added IFW as a substitute.

Reply:
I removed QtIFW. NSIS now works